src/index.html

```diff
@@ -1,13 +1,72 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles.css">
+  </head>
+  <body>
+    <section class="card" data-qa="card">
+      <img
+        class="card__img"
+        src="images/imac.jpeg"
+        width="160"
+        height="134"
+        alt="Macbook"
+      >
+
+      <div class="card__info">
+        <p>
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </p>
+
+        <p>
+          Код товара: 195434
+        </p>
+      </div>
+
+      <div class="card__rating">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+
+          <div class="stars__star"></div>
+
+          <div class="stars__star"></div>
+
+          <div class="stars__star"></div>
+
+          <div class="stars__star"></div>
+        </div>
+
+        <p class="card__review">
+          Отзывов: 5
+        </p>
+      </div>
+
+      <div class="card__price">
+        <p>
+          Цена:
+        </p>
+
+        <p>
+          69 999 грн
+        </p>
+      </div>
+
```

Comment:
You should set this paragraphs as elements of card block too. Add classes to them and use this classes in css to style.